qcut/docs/issues/timeline-video-disappear-after-close/consolev4.md

```
@@ -0,0 +1,104 @@
+ 📍 Original source: media-store-display
+blob-manager.ts:83   🔄 Requested by: VideoPlayer
+blob-manager.ts:84   📊 Ref count: 2
+video-player.tsx:187 [VideoPlayer] Using blob URL for 0d9c8747-4c26-a3e8-65a3-13859c652b4d: blob:app://./9ad3547a-5215-498f-aee9-ba107f741d0c
+video-player.tsx:264 [VideoPlayer] ✅ Video loaded: 0d9c8747-4c26-a3e8-65a3-13859c652b4d
+html2canvas.esm.js:5347 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
+s.createCanvasClone @ html2canvas.esm.js:5347
+html2canvas.esm.js:5347 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
+s.createCanvasClone @ html2canvas.esm.js:5347
+video-player.tsx:328 [VideoPlayer] ▶️ Video ready to play: 0d9c8747-4c26-a3e8-65a3-13859c652b4d
+timeline-track.tsx:696 {"message":"Drop event started in timeline track","dataTransferTypes":["application/x-media-item"],"trackId":"4ebac1d4-e8a6-40e9-a941-4133adc23420","trackType":"media"}
+timeline-track.tsx:938 [TimelineTrack] Processing media item drop: Object
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)